packages/mosip-mock/README.md

```diff
@@ -2,10 +2,22 @@
 
 This simple mock is used to simulate the MOSIP API for local development.
 
-## Usage
+## Mock NIDs
 
-1. Run the mock server:
+The mock considers every NID to be legitimate **EXCEPT**:
 
-```bash
-yarn start
-```
+- `0000000000`
+- `1111111111`
+- `2222222222`
+- `3333333333`
+- `4444444444`
+- `5555555555`
+- `6666666666`
+- `7777777777`
+- `8888888888`
+- `9999999999`
```

Comment:
FYI @tahmidrahman-dsi , know you amended some valid NID's but moved those to mosip-mock and it already had these invalid ones so opted for these.